cmd/istioctl/main.go

```diff
@@ -299,15 +306,15 @@ func readInputs() ([]inputDoc, error) {
 			return nil, fmt.Errorf("could not encode Spec: %v", err)
 		}
 
-		schema, ok := model.IstioConfig[v.Type]
+		ischema, ok := model.IstioConfig[v.Type]
 		if !ok {
 			return nil, fmt.Errorf("unknown spec type %s", v.Type)
 		}
-		rr, err := schema.FromJSON(string(byteRule))
+		rr, err := ischema.FromJSON(string(byteRule))
 		if err != nil {
 			return nil, fmt.Errorf("cannot parse proto message: %v", err)
 		}
-		glog.V(2).Info(fmt.Sprintf("Parsed %v %v into %v %v", v.Type, v.Name, schema.MessageName, rr))
+		glog.V(2).Info(fmt.Sprintf("Parsed %v %v into %v %v", v.Type, v.Name, ischema.MessageName, rr))
```

Comment:
You can use `glog.V(2).Infof` directly and do away with the extra call to `fmt.Sprintf`. I realize this code was here before, but it would be good to clean-up.